tests/GeniusYield/Test/Providers/Mashup.hs

```diff
@@ -38,7 +38,12 @@ providersMashupTests configs =
            delayBySecond
            slotConfig' <- gyGetSlotConfig provider
            pure (protocolParams, systemStart, (show mode, interpreter), stakePools, slotConfig')
-        assertBool "Parameters are not all equal" $ all (== head paramsList) (tail paramsList)
+        assertBool "Parameters are not all equal" $ allEqual paramsList
+    , testCase "Stake address info" $ do
+        saInfos <- forM configs $ \config -> withCfgProviders config mempty $ \GYProviders {..} -> do
+          delayBySecond
+          gyGetStakeAddressInfo $ unsafeStakeAddressFromText "stake_test1upx0fuqcjqs4h5vp687d8j2cng4y5wkmelc6wzm5szq04qsm5d0l6"
+        assertBool "Stake address info are not all equal" $ allEqual saInfos
```

Comment:
Note that CI providers mashup test (which runs all of providers functions and checks for equality against different providers output) does not include local node - kupo provider but it was locally checked `gyGetStakeAddressInfo` indeed returns same value for this test case.